docs/BasicCommunication/PolicyUpdate/index.md

```diff
@@ -12,53 +12,70 @@ Purpose
+    * Ignition cycles since previous successful PTU (``"exchange_after_x_ignition_cycles"`` value in local PT);
+    * The current date is "24 hours prior to module's certificate expiration date":   
+        * the triggers for checking the cert expiration status are:   
+            * ignition on
+            * TLS handshake
+        * in case module's certificate in policies is expired or invalid, the TLS handshake will fail.    
+        * in case TLS handshake fails because module's certificate is expired or invalid, the "count_of_TLS_errors" must not be incremented;   
+        * current model of backend implementation: certificate is always present in PTU (meaning, each successfull request for policies update would bring a certificate in updated PT).
+    * New application (that is, not-yet existing in local PT) registration;
+  3. Parameters values origin:
+    * ``file`` - is the path to the Snapshot of local PolicyTable (Snapshot PT final destination is Policies Server)
+    * ``timeout`` - value taken from ``"timeout_after_x_seconds"`` field of local PT
+    * ``retry`` - array of values from ``"seconds_between_retries"`` field of local PT. SDL handles the PTU retry sequence (re-requesting update if fails to receive during timeout) by itself.
+  4. When SDL is built with EXTERNAL_PROPRIETARY flow, SDL _PoliciesManager_ must change the status to “UPDATING” and notify HMI with OnStatusUpdate("UPDATING") right after SnapshotPT is sent out to to mobile app via OnSystemRequest() RPC.   
 
 !!!
 
 #### Parameters
 
-|Name|Type|Mandatory|Additional|
-|:---|:---|:--------|:---------|
-|file|String|true|minlength: 1<br>maxlength: 255|
-|timeout|Integer|true|minvalue: 0<br>maxvalue: 65535|
-|retry|Integer|true|array: true<br>minsize: 1<br>maxsize: 5<br>minvalue: 0<br>maxvalue: 65535|
+|Name|Type|Mandatory|Additional|Description|
+|:---|:---|:--------|:---------|:----------|
+|file|String|true|minlength: 1<br>maxlength: 255|Location of policy table snapshot. It’s defined in smartDeviceLink.ini as  “PathToSnapshot” parameter|
+|timeout|Integer|true|minvalue: 0<br>maxvalue: 65535|Send attempt timeout in seconds, it’s a value from the Policy Table.|
+|retry|Integer|true|array: true<br>minsize: 1<br>maxsize: 5<br>minvalue: 0<br>maxvalue: 65535|Array of delays to wait after failed atempts, it’s a value from the Policy Table|
 
 ### Response
 
-!!! must
+!!! MUST
 
-1. Respond with ``SUCCESS`` resultCode to continue the PTU flow.
+  1. Respond with ``SUCCESS`` resultCode to continue the PTU flow.
 
 !!!
 
 #### Parameters
 
 This RPC has no additional parameter requirements
 
+### Sequence Diagrams   
+|||
+_EXTERNAL_PROPRIETARY_ Policy Table Update Flow
+![EXTERNAL proprietary](./assets/diagram_PolicyUpdate_external_proprietary.png)   
+|||
+|||
+BC.PolicyUpdate in _PROPRIETARY_ Policy Table Update Flow
+![Proprietary PTU](./assets/PolicyUpdate_in_Proprietary_PTU_flow.png)  
+|||
```

Comment:
@DrachenkoAnastasiia These blocks are not formatting correctly. I'm not entirely certain, but it may be that these block can't handle special formatting. I would suggest trying to remove all _italic_ word formatting from these blocks.